src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -18,11 +18,17 @@ public void GetAccessControl_DirectoryInfo_InvalidArguments()
         [Fact]
         public void GetAccessControl_DirectoryInfo_ReturnsValidObject()
         {
-            DirectoryInfo directoryInfo = new DirectoryInfo("\\");
+            string directory = Path.GetTempPath() + Guid.NewGuid();
```

Comment:
A more concise pattern, which chooses the name for you, is 

```
            using (var temp = new TempDirectory())
            {
                   // use whtaever temp is
            }
```

example https://github.com/dotnet/corefx/blob/a8dce7c1c905e994c9519e0f26b0497832942a10/src/System.Configuration.ConfigurationManager/tests/Mono/ConfigurationErrorsExceptionTest.cs#L118